src/pages/ContentScripts/index.ts

```diff
@@ -18,8 +18,10 @@ import { inject2Perceptor, Perceptor } from '../../Perceptor';
 
 import { loadSettings } from '../../utils/settings';
 
+
 import './index.css';
```

Comment:
Well, I think this line should be removed.

Reply:
yes done👍